code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java

```diff
@@ -137,97 +98,84 @@ void handleTrackRequest(final Event event) {
 	 */
 	void handleRulesEngineResponse(final Event event) {
 		if (event == null) {
```

Comment:
Given that `handleRulesEngineResponse` is an implementation of `ExtensionEventListener`, the incident `event` is   non null by definition. IMO you can get rid of the null check on event and annotate with @NonNull

Reply:
The null guard exists in the definition of [`notify(event: Event)`](https://github.com/adobe/aepsdk-core-android/blob/d33be1e9eb593de0837a91fe7b4a265b1ecdbeb4/code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventListenerContainer.kt#L63) which invokes listeners. Since the notify method only accepts a non null `event` ( a null accepting method will look like `notify(event: Event?`) ), it is acceptable to get rid of the if (event == null) inside _all_ the listener handlers that exist in this class.